pkgs/tools/audio/beets/common.nix

```diff
@@ -143,5 +143,12 @@ python3Packages.buildPythonApplication rec {
     license = licenses.mit;
     maintainers = with maintainers; [ aszlig doronbehar lovesegfault pjones ];
     platforms = platforms.linux;
+    # ModuleNotFoundErrors during build:
+    # - requests_oauthlib
+    # - discogs_client
+    # - mpd
+    # - xdg
+    # - flask
+    broken = true;
```

Comment:
This is incorrect, `beets` builds and runs fine (with current master).

Reply:
Hmm, this definitely fails: `nix build .#beetsPackages.beets-minimal`, but `nix build .#beets` indeed works fine.

Reply:
Gonna drop the commit for now, but have to circle back to it in the end if it doesn't get fixed. I agree that it should be done differently though.

Reply:
#173892